src/Avalonia.X11/X11Window.cs

```diff
@@ -984,20 +996,31 @@ public void SetCursor(ICursorImpl? cursor)
         
         public PixelPoint Position
         {
-            get => _position ?? default;
+            get
+            {
+                var extents = GetFrameExtents();
+
+                if(extents == null || _position == null)
```

Comment:
Extents aren't guaranteed to be provided by the window manager

Reply:
Ok. Done.